src/SpendPermissionManager.sol

```diff
@@ -301,14 +303,11 @@ contract SpendPermissionManager is EIP712 {
     }
 
     /// @notice Approve a spend permission batch via a signature from the account.
-    ///
-    /// @dev Compatible with ERC-6492 signatures including side effects (https://eips.ethereum.org/EIPS/eip-6492).
-    /// @dev Does not enforce uniqueness of permissions within a batch, allowing duplicate idempotent approvals.
-    ///
-    /// @param spendPermissionBatch Details of the spend permission batch.
-    /// @param signature Signed approval from the user.
-    ///
-    /// @return allApproved True if all spend permissions in the batch are approved and not revoked.
+    /// @dev Compatible with ERC-6492 signatures. Does not enforce uniqueness of permissions 
+    ///      within a batch, allowing duplicate idempotent approvals.
+    /// @param spendPermissionBatch Details of the spend permission batch
+    /// @param signature Signed approval from the user
+    /// @return allApproved True if all spend permissions in the batch are approved and not revoked
```

Comment:
https://github.com/coinbase/solidity-style-guide?tab=readme-ov-file#3-newlines-between-tag-types
think we want to keep the new-lines

Reply:
good catch, just gave this a once-over for consistency across all functions again